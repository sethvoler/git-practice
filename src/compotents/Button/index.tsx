import React from 'react';

const Button: React.FC = ({ children }) => <button>{children}</button>;

export default Button;
