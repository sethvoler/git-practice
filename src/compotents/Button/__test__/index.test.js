import React from 'react';
import Button from '../index';
import { snapshotTest } from '@test/index';

const testObj = {
  name: 'Button 组件',
  desc: '正确渲染',
  component: <Button>按钮</Button>,
};

snapshotTest(testObj);
