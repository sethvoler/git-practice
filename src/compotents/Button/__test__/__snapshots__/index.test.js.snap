// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 组件 Button 组件 正确渲染 1`] = `
<button>
  按钮
</button>
`;
